File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        single_digits = [num for num in nums if num < 10]
        double_digits = [num for num in nums if num >= 10]

        sum_single = sum(single_digits)
        sum_double = sum(double_digits)

        total_sum = sum(nums)

        sum_bob_single = total_sum - sum_single
        sum_bob_double = total_sum - sum_double

        if sum_single > sum_bob_single or sum_double > sum_bob_double:
            return True
        else:
            return False

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var canAliceWin = function(nums) {
    let singles = 0;
    let doubles = 0;
    for (let num of nums) {
        if (num.toString().length === 1) {
            singles += num;
        }
        else doubles += num;
    }
    return singles !== doubles;
};